oktapam/constants/descriptions/data_sources.go

```diff
@@ -16,17 +16,17 @@ var (
 	// Features In Progress
 
 	// Fetch
-	SourceADConnections = "A list of ASA AD Connections associated with an ASA Team."
-	SourceGateways      = fmt.Sprintf("Returns a list of all ASA Gateways connected to the ASA Team specified in the OKTAPAM_TEAM environment variable. %s", LinkGateway)
-	SourceGroup         = fmt.Sprintf("Returns a previously created ASA Group. %s", LinkGroup)
-	SourceProject       = fmt.Sprintf("Returns a previously created ASA Project. %s", LinkProject)
-	SourceProjectGroup  = fmt.Sprintf("Returns a previously created ASA Group assigned to a given ASA Project. %s", LinkProjectGroup)
-	SourceTeamSettings = fmt.Sprintf("Returns Team-level settings for a specific Team, such as authentication and enrollment details")
+	SourceADConnections          = "A list of ASA AD Connections associated with an ASA Team."
```

Comment:
Mirroring earlier comment, if we remove ASA from the description text will the descriptions still work? Or if these descriptions contain ASA can we ensure that they are referenced only from objects that are ASA specific?

Reply:
I think better to remove ASA.